### ISSUE TITLE
frame_object_diff - bitset seriaziation issue (Trac #957)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#951]
cweaver: Should t.count() be t.size()? http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp#L104 

Very likely.  Generate a unit test that breaks because of this, then fix it.

Migrated from https://code.icecube.wisc.edu/ticket/957
```json
{
    "status": "closed", 
    "changetime": "2015-07-07T22:42:37", 
    "description": "cweaver: Should t.count() be t.size()? http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp#L104 \n\nVery likely.  Generate a unit test that breaks because of this, then fix it.", 
    "reporter": "david.schultz", 
    "cc": "david.schultz", 
    "resolution": "fixed", 
    "_ts": "1436308957786183", 
    "component": "combo reconstruction", 
    "summary": "frame_object_diff - bitset seriaziation issue", 
    "priority": "major", 
    "keywords": "bitset", 
    "time": "2015-05-01T03:37:45", 
    "milestone": "", 
    "owner": "cweaver", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150519T21:50:05: 
* david.schultz changed priority from "critical" to "major"
* david.schultz changed milestone from "" to "offline V15-05-00"

**Comment by icecube-trac** [comment]
Trac update at 20150519T22:29:21: 
* cweaver changed resolution from "" to "fixed"
* cweaver changed status from "new" to "closed"
* cweaver commented:

> In changeset 132689:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="132689"
> Add more tests to cover dynamic bitsets. 
> Fix a bug in dynamic bitset sizes. Closes #951.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150707T22:42:37: 
* olivas changed milestone from "offline V15-05-00" to ""
* olivas commented:

> Milestone offline V15-05-00 deleted
